src/ui/CMakeLists.txt

```diff
@@ -162,10 +164,22 @@ else() # native build
             yaml-cpp::yaml-cpp
             core
             client
+            services
             app_header
             utils
             ui_assets
             sample_dashboards
             fonts)
   target_compile_definitions(${target_name} PRIVATE BLOCKS_DIR="${GNURADIO_PREFIX}/share/gnuradio/grc/blocks/")
 endif()
+
+target_link_libraries(${target_name} PRIVATE function2::function2)
+target_include_directories(${target_name} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../service/acquisition) # for daq_api.hpp
```

Comment:
The target only exists in the native branch of the if above -> should be moved inside. Also the function2 dependency is already included there and should not be necessary. 

The daq api include path will be picked up if you build from the repo root by the od_acquisition dependency, but the target does not exist if we only build the ui separately and I think it doesn't hurt to also add the path explicitly.